hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/CompactSplit.java

```diff
@@ -229,8 +229,10 @@ private synchronized void requestSplit(final Region r, byte[] midKey) {
    */
   private synchronized void requestSplit(final Region r, byte[] midKey, User user) {
     if (midKey == null) {
-      LOG.debug("Region " + r.getRegionInfo().getRegionNameAsString()
-        + " not splittable because midkey=null");
+      if (LOG.isDebugEnabled()) {
```

Comment:
Are these logging changes related?

Reply:
no, it is not related, just modified by the way.